src/main/java/sirius/biz/web/BizController.java

```diff
@@ -127,29 +127,36 @@ protected void assertTenant(@Nonnull String tenantId) {
         }
     }
 
-    private HandledException invalidTenantException() {
+    protected HandledException invalidTenantException() {
         return Exceptions.createHandled().withNLSKey("BizController.invalidTenant").handle();
     }
 
     /**
      * Enusures or establishes a parent child relation.
      * <p>
-     * For new entities (owner), the given reference is initialized with the given entity. For existing entities
+     * For new entities (child), the given reference is initialized with the given entity. For existing entities
      * it is verified, that the given reference points to the given entity.
      *
-     * @param owner  the entity which contains the reference
-     * @param ref    the reference which is either filled or verified that it points to <tt>entity</tt>
-     * @param entity the entity the reference must point to
-     * @param <E>    the generic type the the entity being referenced
-     * @param <I>    the type of the id column of E
+     * @param child        the entity which contains the reference
+     * @param ref          the reference which is either to be filled or verified that it points to <tt>parentEntity</tt>
+     * @param parentEntity the entity the reference must point to
+     * @param <E>          the generic type the the parent being referenced
+     * @param <I>          the type of the id column of E
      * @throws sirius.kernel.health.HandledException if the entities do no match
      */
-    protected <I, E extends BaseEntity<I>> void setOrVerify(BaseEntity<?> owner, BaseEntityRef<I, E> ref, E entity) {
-        if (!Objects.equals(ref.getId(), entity.getId())) {
-            if (owner.isNew()) {
-                ref.setValue(entity);
+    protected <I, E extends BaseEntity<I>> void setOrVerifyParent(BaseEntity<?> child,
```

Comment:
this isn't necessarrly a parent child relation so i'd rather keep the neutral name

Reply:
The first javadoc line describing the method was "Enusures or establishes a parent child relation." how would you replace the javadoc?

Reply:
Maybe, "Properly maintains a reference to an entity with write-once semantic"
and also add  a @see BaseEntityRef#writeOnceOn and encourage using it